pkg/registry/metadata.go

```diff
@@ -4,31 +4,23 @@ import (
 	"fmt"
 
 	k0sv1beta1 "github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
-	ectypes "github.com/replicatedhq/embedded-cluster-kinds/types"
 	"sigs.k8s.io/yaml"
 )
 
-func getSeaweedfsNamespaceFromMetadata(metadata *ectypes.ReleaseMetadata) (string, error) {
-	chart, err := getSeaweedfsChartFromMetadata(metadata)
-	if err != nil {
-		return "", fmt.Errorf("get seaweedfs charts settings from metadata: %w", err)
+func getSeaweedfsS3SecretNameFromHelmExtension(ext k0sv1beta1.HelmExtensions) (string, error) {
```

Comment:
I guess I should have mentioned this before but I would not be against just hardcoding these

Or having embedded-cluster's installer use a constant that it imports from here